cypress/integration/core-opensearch-dashboards/opensearch-dashboards/mds_dashboard_sanity_test_spec.js

```diff
@@ -0,0 +1,353 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  CommonUI,
+  MiscUtils,
+} from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
+
+const commonUI = new CommonUI(cy);
+const miscUtils = new MiscUtils(cy);
+const baseURL = new URL(Cypress.config().baseUrl);
+// remove trailing slash
+const path = baseURL.pathname.replace(/\/$/, '');
+
+const disableLocalCluster = !!Cypress.env('DISABLE_LOCAL_CLUSTER');
+
+if (Cypress.env('DATASOURCE_MANAGEMENT_ENABLED')) {
+  describe('dashboard data source sample data validation', () => {
+    before(() => {});
+
+    after(() => {});
+
```

Comment:
```suggestion
    before(() => {});

    after(() => {});
```
Useless code I suppose?

Reply:
removed